tests/installation/release_notes_from_url.pm

```diff
@@ -19,7 +19,14 @@ sub run {
     enter_cmd "zgrep -oh \"Got release notes.*\" /var/log/YaST2/y2log*";
     assert_screen [qw(got-releasenotes-RPM got-releasenotes-URL)];
     unless (match_has_tag 'got-releasenotes-URL') {
-        record_soft_failure('bsc#1190711 - Release notes source does NOT match expectations or not found in YaST logs, expected source: URL');
+        my $not_public = script_run('grep -r RC /README.BETA');
+        my $error_msg = 'Release notes source does NOT match expectations or not found in YaST logs, expected source: URL';
+        unless (get_var('BETA') || $not_public) {
```

Comment:
good practice is not avoid using `unless` in general but in perl is handy sometimes, but not with more than one condition. It is always better replace by `if`.

Reply:
I don't understand this condition, so if it is not beta and it is public then we die. There is RC, PublicRC and GMC, but I guess PublicRC is the same than RC in this file? I'm not sure your intention here.

Reply:
ahm that you identify GMC like not beta...uff messy to read definitively, I will think about it and come back to you.

Reply:
The problem is that I am not aware of another way to check if the build is GMC